src/helpers/block_requests.rs

```diff
@@ -98,102 +98,133 @@ pub fn find_common_ancestor<N: Network>(canon: &LedgerState<N>, block_locators:
     Ok((maximum_common_ancestor, first_deviating_locator))
 }
 
+/// A case annotation enum for the block request handler.
+#[derive(Debug, PartialEq, Eq)]
+pub(crate) enum Case {
+    /// Case 0 - The common ancestor is greater than the latest block height:
+    ///     - Abort. This is an internal error that needs to be remedied promptly.
+    Zero,
```

Comment:
note: this case - while pretty trivial - doesn't have a corresponding test

Reply:
Adding a test case for 0 now

Reply:
Done